### ISSUE TITLE
Valve whitelist for secured stats

### ISSUE THREAD

**Issue by jimmydorry** [GetDotaStats/site#293]
Find the old issue and merge

**Comment by jimmydorry** [comment]
http://media.steampowered.com/apps/sdr/network_config.json

http://dev.dota2.com/showthread.php?t=284133

Use the officially sanctioned list

**Comment by MNoya** [comment]
Valve communication 😮

**Comment by jimmydorry** [comment]
Indeed. It would be too bad though if you were just starting and were looking for a list of IPs to trust. It's no longer going to be something you can find purely in dota client files.
